keps/349-all-or-nothing/README.md

```diff
@@ -254,6 +254,11 @@ the Kueue's Job reconciller already fetches the latest Job object at the
 beginning of the `Reconcile` function. Also note, there is no new condition
 introduced to the Job API.
 
+Note that, we don't take failed pods directly into the computation for verifying
+the `PodsReady` condition. However, we would eliminate a buggy workload based on
```

Comment:
It sounds like kueue will do something extra about this situation. But in reality we just wait for the job to fail.

Reply:
yeah, rephrased slightly